[DOCS] update action docs to include custom field example (#10987)
great-expectations/great_expectations: docs/docusaurus/docs/core/trigger_actions_based_on_results/create_a_custom_action.md

   ```python title="Python" name="docs/docusaurus/docs/core/trigger_actions_based_on_results/_examples/create_a_custom_action.py - set type" 
   ```

> 3.  Optionally, add any additional fields your action requires at runtime. Actions are built on Pydantic models. Define the field name a class-level attribute on your Action, and annotate it with the correct type. When you instantiate the Action, pass the field value into the Action init method. Your Action will have access to these values within your custom `run` method via `self.<MY FIELD NAME>`.

Reviewer: suggestion addresses a few style guide and grammar things

```suggestion
3.  Optional. Add any additional fields your Action requires at runtime. Actions are built on Pydantic models. Define the field name as a class-level attribute on your Action, and annotate it with the correct type. When you instantiate the Action, pass the field value into the Action `init` method. Your Action will have access to these values within your custom `run` method through `self.<MY_FIELD_NAME>`.
```

>    ```python title="Python" name="docs/docusaurus/docs/core/trigger_actions_based_on_results/_examples/create_a_custom_action.py - add custom fields"

Reviewer: I think there needs to be a blank line here for the code block to indent properly

<img width="809" alt="Screenshot 2025-03-03 at 8 01 24 AM" src="https://github.com/user-attachments/assets/dd54bc6c-e64a-4ae4-84af-568af0fc4801" />

```suggestion

   ```python title="Python" name="docs/docusaurus/docs/core/trigger_actions_based_on_results/_examples/create_a_custom_action.py - add custom fields"
```
